src/components/PostsList.tsx

```diff
@@ -1,85 +1,92 @@
 import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[];
+  selectedPost: Post | null;
+  isCommentsListtHidden: boolean;
+  openDetails: (post: Post) => void;
+  closeCommentList: React.Dispatch<React.SetStateAction<boolean>>;
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  openDetails,
+  selectedPost,
+  closeCommentList,
+  isCommentsListtHidden,
```

Comment:
There is a typo in the name of the `isCommentsListtHidden` prop. It should be `isCommentsListHidden`.